hack/lib/util.sh

```diff
@@ -48,6 +48,32 @@ kube::util::wait_for_url() {
   return 1
 }
 
+# returns a random port
+kube::util::get-random-port() {
```

Comment:
nit: maybe use underscores instead of dashes in the name?

this file is horribly inconsistent, but the Google shell style guide recommends underscores over dashes.